src/components/Result.js

```diff
@@ -31,6 +32,60 @@ function Result() {
   let score1 = localStorage.getItem("score1");
   let score2 = localStorage.getItem("score2");
   let numberOfPlayers = localStorage.getItem('players');
+
+
+  useEffect(()=>{
```

Comment:
avoid the duplication of the same code

Reply:
Created Another file added logic in that file and using it wherever it required